holoviews/plotting/bokeh/element.py

```diff
@@ -55,10 +57,6 @@
 
 class ElementPlot(BokehPlot, GenericElementPlot):
 
-    callbacks = param.ClassSelector(class_=Callbacks, doc="""
-        Callbacks object defining any javascript callbacks applied
-        to the plot.""")
-
```

Comment:
Great to get rid of this! Though we mustn't forget to mention its removal in the CHANGELOG.